tests/test_off_grid_sensors.py

```diff
@@ -0,0 +1,82 @@
+from jwave.geometry import _bli_function, BLISensors, Domain, FourierSeries
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize("n_grid", [(100, ), (101, )])
+def test_bli_function(n_grid=100):
```

Comment:
The current tests fail here: https://github.com/ucl-bug/jwave/actions/runs/5381874402/jobs/9785603397?pr=192#step:5:37

I think it is just because one can't put a default value for the arguments of a parametrized test, so probably changing to `def test_bli_function(ngrid):` will be enough